mmv1/products/dataproc/SessionTemplate.yaml

```diff
@@ -0,0 +1,235 @@
+    primary_resource_name: 'fmt.Sprintf("projects/%s/locations/%s/sessionTemplates/%s", envvar.GetTestProjectFromEnv(), envvar.GetTestRegionFromEnv(), fmt.Sprintf("tf-test-session-templates-jupyter%s", context["random_suffix"]))'
+    vars:
+      name: 'jupyter-session-template'
+      subnetwork_name: 'default'
+      prevent_destroy: 'true'
+    test_env_vars:
+      project_name: 'PROJECT_NAME'
+    test_vars_overrides:
+      'subnetwork_name': 'acctest.BootstrapSubnetWithFirewallForDataprocBatches(t, "jupyer-session-test-network", "jupyter-session-test-subnetwork")'
+      'prevent_destroy': 'false'
+    ignore_read_extra:
+      - 'runtime_config.0.properties'
+  - name: 'dataproc_session_templates_jupyter_full'
+    primary_resource_id: 'dataproc_session_templates_jupyter_full'
+    primary_resource_name: 'fmt.Sprintf("projects/%s/locations/%s/sessionTemplates/%s", envvar.GetTestProjectFromEnv(), envvar.GetTestRegionFromEnv(), fmt.Sprintf("tf-test-session-templates-jupyter%s", context["random_suffix"]))'
+    vars:
+      name: 'jupyter-session-template'
+      subnetwork_name: 'default'
+      prevent_destroy: 'true'
+      kms_key_name: 'example-key'
+      bucket_name: 'dataproc-bucket'
+    test_env_vars:
+      project_name: 'PROJECT_NAME'
+    test_vars_overrides:
+      'subnetwork_name': 'acctest.BootstrapSubnetWithFirewallForDataprocBatches(t, "jupyer-session-test-network", "jupyter-session-test-subnetwork")'
+      'prevent_destroy': 'false'
+      'kms_key_name': 'acctest.BootstrapKMSKeyWithPurposeInLocationAndName(t, "ENCRYPT_DECRYPT", "us-central1", "tf-bootstrap-dataproc-session-template-key1").CryptoKey.Name'
+    ignore_read_extra:
+      - 'runtime_config.0.properties'
+  - name: 'dataproc_session_templates_spark_connect'
+    primary_resource_id: 'example_session_templates_spark_connect'
+    primary_resource_name: 'fmt.Sprintf("projects/%s/locations/%s/sessionTemplates/%s", envvar.GetTestProjectFromEnv(), envvar.GetTestRegionFromEnv(), fmt.Sprintf("tf-test-session-templates-jupyter%s", context["random_suffix"]))'
+    vars:
+      name: 'sc-session-template'
+      subnetwork_name: 'default'
+      prevent_destroy: 'true'
+    test_env_vars:
+      project_name: 'PROJECT_NAME'
+    test_vars_overrides:
+      'subnetwork_name': 'acctest.BootstrapSubnetWithFirewallForDataprocBatches(t, "spark-connect-session-test-network", "spark-connect-session-test-subnetwork")'
+      'prevent_destroy': 'false'
+    ignore_read_extra:
+      - 'runtime_config.0.properties'
+parameters:
+  - name: 'location'
+    type: String
+    description: |
+      The location in which the session template will be created in.
+    url_param_only: true
+    immutable: true
+properties:
+  - name: 'name'
+    type: String
+    description: The resource name of the Feature Online Store. This value may be up to 60 characters, and
+      valid characters are [a-z0-9_]. The first character cannot be a number.
```

Comment:
```suggestion
    description: The resource name of the session template. In the format:
    'projects/{project}/locations/{location}/sessionTemplates/{template_id}'
```

Reply:
can we make sure there aren't any other Feature Online Store descriptions copied over?